release_notes/v2.1.0.md

```diff
@@ -0,0 +1,150 @@
+v2.1.0 Release Notes - April XX, 2020
+========================================
+
+What's New in Hyperledger Fabric v2.1
+-------------------------------------
+
+**FAB-17357: Add endorser metric for simulation failure**
+
+Added metric endorser_proposal_simulation_failures.
+
+
+**FAB-14761: Limit concurrent requests to endorser and deliver services**
+
+Limits can now be placed on the number of endorser and deliver requests
+that a peer will process at any one time. If a peer is already processing its
+limit of requests, subsequent requests in excess of the limit will return an error,
+and the client will need to retry the call. The limits are configured using the
+following core.yaml properties:
+- peer.limits.concurrency.endorserService
+- peer.limits.concurrency.deliverService
+
+**FAB-17463: Allow peer to override implicit collection dissemination properties**
+
+The following config properties have been added to peer's core.yaml
+for implicit private data collection dissemination:
+- peer.gossip.pvtData.ImplicitCollectionDisseminationPolicy.requiredPeerCount
+- peer.gossip.pvtData.ImplicitCollectionDisseminationPolicy.maxPeerCount
+
+When a peer endorses a transaction that writes to its own organization's
+implicit private data collection, the new properties will dictate how
+many other peers in the organization the endorsing peer will attempt to
+disseminate to (maxPeerCount), and how many peers must acknowledge receipt
+of the private data before endorsement succeeds (requiredPeerCount).
+These properties are applicable to all channels the peer has joined. The implication
+is that requiredPeerCount has to be smaller than the number of peers in a channel
+that has the lowest numbers of peers from the organization.
+
+**FAB-17299: Support collection level endorsement policies for discovery**
+
+* PoliciesByChaincode now returns one InquireablePolicy per collection,
+either the collection's endorsement policy if one exists or the
+chaincode policy by default
+* chaincode.Metadata now stores CollectionPolicies for lifecycle
+namespace chaincodes that have collection endorsement policies
+
+
+**Discover CLI now supports SEC 1 formatted private keys**
+Private keys that are generated with openssl ecparam (SEC 1 format) are now supported
+with the discover CLI, in addition to the PKCS8 private keys that were already supported.
+
+
+**Dependency updates**
+
+- Bump docker images to Alpine 3.11.
+- Bump Go to 1.14.1.
+- Bump Go grpc to 1.28.0.
+
+
+Fixes
+-----
+
+**FAB-17458: Rolling upgrade: Different endorsement results on v1.4.x and v2.0.0 peers (FIX ALSO IN V2.0.1)**
```

Comment:
For the fixes also in v2.0.1 (release-2.0 branch), I think I'll remove from the v2.1 release notes and just mention that all fixes in v2.0.1 are also merged into v2.1.